store/commitment/store.go

```diff
@@ -78,8 +82,9 @@ func (c *CommitStore) WriteBatch(cs *store.Changeset) error {
 func (c *CommitStore) WorkingCommitInfo(version uint64) *proof.CommitInfo {
 	storeInfos := make([]proof.StoreInfo, 0, len(c.multiTrees))
 	for storeKey, tree := range c.multiTrees {
+		bz := internal.UnsafeStrToBytes(storeKey)
```

Comment:
here it's not safe to use UnsafeStrToBytes and we should instead yield a copy doing ``bz := []byte(storeKey)``